presto-native-execution/src/test/java/com/facebook/presto/nativeworker/NativeQueryRunnerUtils.java

```diff
@@ -282,7 +282,7 @@ public static void createCustomer(Session session, QueryRunner queryRunner)
     {
         if (!queryRunner.tableExists(session, "customer")) {
             queryRunner.execute(session, "CREATE TABLE customer AS " +
-                    "SELECT custkey, name, address, nationkey, phone, acctbal, comment, mktsegment " +
+                    "SELECT custkey, name, address, nationkey, phone, acctbal, mktsegment, comment " +
```

Comment:
@anandamideShakyan : Don't entirely follow why this fixed the problem ? Can you elaborate.

Reply:
We have two query runners actual(native) and expected(h2). When we generate the table for native query runner these two columns were swapped out which was causing the column mismatch error. I looked at the table schema for java and made the changes in the native one.

Reply:
The columns are given the same name as the source column name, so the order doesn't really matter here.

@pramodsatya, @pdabre12 : Can you help investigate further.